### PR TITLE
VRF-664:add Slack notification for VRF v2 Plus WASP test

### PR DESCRIPTION
**Pull request by iljapavlovs**

### CODE REVIEW

File: .github/workflows/on-demand-vrfv2plus-performance-test.yml [hunk 1]
```diff
@@ -80,6 +80,8 @@ jobs:
       REF_NAME: ${{ github.head_ref || github.ref_name }}
       CHAINLINK_IMAGE: ${{ inputs.chainlinkImage }}
       CHAINLINK_VERSION: ${{ inputs.chainlinkVersion }}
+      SLACK_API_KEY: ${{ secrets.QA_SLACK_API_KEY }}
+      SLACK_CHANNEL: C0629GXEB0D
```
Comment (tateexon) [inline .github/workflows/on-demand-vrfv2plus-performance-test.yml:84]: Is this ok to not be a secret?

Comment (iljapavlovs) [reply .github/workflows/on-demand-vrfv2plus-performance-test.yml:84]: our guess is that its not a secret since it's just an identifier

Comment (jmank88) [reply .github/workflows/on-demand-vrfv2plus-performance-test.yml:84]: FWIW I was able to use channel id here:
https://github.com/smartcontractkit/chainlink/blob/c1fc9d293ecc5822ec06aa15797d32d8cc10ad83/.github/workflows/ci-core.yml#L107

Comment (iljapavlovs) [reply .github/workflows/on-demand-vrfv2plus-performance-test.yml]: end up adding new secret - `QA_VRF_SLACK_CHANNEL `

**Review by tateexon — APPROVED** [review]

**Pull request merged**